src/EndpointManager.sol

```diff
@@ -109,8 +109,14 @@ abstract contract EndpointManager is IEndpointManager, OwnableUpgradeable, Reent
         // normalize amount decimals
         uint256 normalizedAmount = normalizeAmount(amount, decimals);
 
-        bytes memory encodedTransferPayload =
-            encodeNativeTokenTransfer(normalizedAmount, _token, recipient, recipientChain);
+        NativeTokenTransfer memory nativeTokenTransfer = NativeTokenTransfer({
+            amount: normalizedAmount,
+            tokenAddress: toWormholeFormat(_token),
+            to: recipient,
+            toChain: recipientChain
+        });
+
+        bytes memory encodedTransferPayload = encodeNativeTokenTransfer(nativeTokenTransfer);
```

Comment:
FYI I had previously removed this based on feedback from @djb15 that the `NativeTokenTransfer` and `EndpointManagerMessage` structs weren't used anywhere else.

I figure it's mostly a style thing -- I don't have a super strong opinion either way. Maybe the nicest style here would be to follow what you wrote on line 123 for `encodeEndpointManagerMessage` -- 
```
bytes memory encodedTransferPayload = encodeNativeTokenTransfer(NativeTokenTransfer(...));
```

Reply:
happy to revisit this later! personally I like that `encode` is called on a struct, make things a bit more symmetrical